pkg/vsphere/extraconfig/basic_test.go

```diff
@@ -357,13 +357,16 @@ func TestInheritenceOfNonPersistence(t *testing.T) {
 
 	type Type struct {
 		Common CommonPersistence `vic:"0.1" scope:"read-write,non-persistent" key:"common"`
+
+		Notes string `vic:"0.1" scope:"hidden" key:"notes"`
 	}
 
 	Struct := Type{
 		Common: CommonPersistence{
 			ID:   "0xDEADBEEF",
 			Name: "Struct",
 		},
+		Notes: "notes",
```

Comment:
Had deliberately left this out to ensure that empty fields didn't get serialized.